.changeset/heavy-cobras-own.md

```diff
@@ -0,0 +1,7 @@
+---
+"@blitzjs/auth": patch
+"@blitzjs/next": patch
+"blitz": patch
+---
+
+Ensure the component has mounted before throwing a redirect with redirectAuthenticatedTo. This will fix the hydration error from react.
```

Comment:
It's better to start with the "what" because that's the most important info to users than the "how".
```suggestion
Fix auth related React hydration errors by not redirecting until after component mount.
```